### PR TITLE
implement arp responder

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kiman@vmware.com>
Co-authored-by: Zaalouk, Adel <adel.zalok.89@gmail.com>

### CODE REVIEW

File: controllers/openflow/flows.go [hunk 1]
```diff
@@ -38,14 +38,17 @@ const (
...
+	tableL2Forwarding    = 40
+	tableL3Rewrites      = 50
+	tableL2Rewrites      = 60
+	tableProxy           = 70
+	tableNAT             = 80
+	tableAudit           = 90
+	tableLocalARP        = 100
+
+	// all pods have a fixed mac addr
+	podMacAddr = "aa:bb:cc:dd:ee:ff"
```
Comment (zanetworker) [inline controllers/openflow/flows.go:51]: this is defined twice (here and veth.go), would a common home for it make sense?

**Review by zanetworker — COMMENTED** [review]
As you mentioned, there are duplicate code from CNI, for now its needed, but maybe we should PR CNI with LinkAttr as a param instead of just name and MTU (but that would be a breaking change for other CNIs though)? 

otherwise /lgtm

File: controllers/openflow/flows.go (hunk 1 above)
Comment (andrewsykim) [reply controllers/openflow/flows.go:51]: A single package/file for 1 const seemed overkill but agree we should revisit this soon

**Comment by andrewsykim** [comment]
Agreed on both comments. Let's see how this pans out and revisit this code, definitely needs some clean up. Created https://github.com/k-vswitch/k-vswitch/issues/11 to follow-up.

**Pull request merged**